packages/smooth_app/lib/cards/product_cards/smooth_product_card.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:smooth_app/generic_lib/widgets/smooth_card.dart';
+
+class SmoothProductBaseCard extends StatelessWidget {
```

Comment:
Better if class name and file name could match.
A `///` comment would be nice too.

Reply:
Ok, I change that